### PR TITLE
docs: add examples for removing headers

### PR DESCRIPTION
**Pull request by SuperSandro2000**
### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/middlewares/headers.md [hunk 1]
```diff
@@ -78,6 +78,8 @@ and responses are stripped of their `X-Custom-Response-Header` header.
 ```yaml tab="Docker"
 labels:
   - "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Script-Name=test"
+  - "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Custom-Request-Header=\"\""
+  - "traefik.http.middlewares.testheader.headers.customResponseHeaders.X-Custom-Response-Header=\"\""
```
Comment (dtomcej) [inline docs/content/middlewares/headers.md:82]:
```suggestion
  - "traefik.http.middlewares.testheader.headers.customresponseheaders.X-Custom-Response-Header=\"\""
```

File: docs/content/middlewares/headers.md [hunk 2]
```diff
@@ -96,17 +98,23 @@ spec:
 
 ```yaml tab="Consul Catalog"
 - "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Script-Name=test"
+- "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Custom-Request-Header=\"\""
+- "traefik.http.middlewares.testheader.headers.customResponseHeaders.X-Custom-Response-Header=\"\""
```
Comment (dtomcej) [inline docs/content/middlewares/headers.md:102]:
```suggestion
- "traefik.http.middlewares.testheader.headers.customresponseheaders.X-Custom-Response-Header=\"\""
```

File: docs/content/middlewares/headers.md [hunk 3]
```diff
@@ -96,17 +98,23 @@ spec:
...
 ```
 
 ```json tab="Marathon"
 "labels": {
   "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Script-Name": "test",
+  "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Custom-Request-Header": "",
+  "traefik.http.middlewares.testheader.headers.customResponseHeaders.X-Custom-Response-Header": "",
```
Comment (dtomcej) [inline docs/content/middlewares/headers.md:109]:
```suggestion
  "traefik.http.middlewares.testheader.headers.customresponseheaders.X-Custom-Response-Header": "",
```

File: docs/content/middlewares/headers.md [hunk 4]
```diff
@@ -96,17 +98,23 @@ spec:
...
 }
 ```
 
 ```yaml tab="Rancher"
 labels:
   - "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Script-Name=test"
+  - "traefik.http.middlewares.testheader.headers.customrequestheaders.X-Custom-Request-Header=\"\""
+  - "traefik.http.middlewares.testheader.headers.customResponseHeaders.X-Custom-Response-Header=\"\""
```
Comment (dtomcej) [inline docs/content/middlewares/headers.md:117]:
```suggestion
  - "traefik.http.middlewares.testheader.headers.customresponseheaders.X-Custom-Response-Header=\"\""
```

**Comment by dtomcej** [comment]
Hello @SuperSandro2000, do you think that you could rebase this PR off of the 2.4 branch? Thanks!

**Comment by SuperSandro2000** [comment]
> Hello @SuperSandro2000, do you think that you could rebase this PR off of the 2.4 branch? Thanks!

Sure. I also changed the base branch to v2.4.

**Review by jbdoumenjou — APPROVED** [review]
LGTM

**Review by rtribotte — APPROVED** [review]
LGTM

**Review by ldez — APPROVED** [review]
LGTM

**Comment by ldez** [comment]
We have an issue with Travis, I will close then reopen the PR

**Pull request merged**